test1.py

```diff
@@ -1,12 +1,14 @@
 import tensorflow as tf
-tf.set_random_seed(777)  # for reproducibility
+tf.set_random_seed(1774)  # for reproducibility
```

Comment:
Why did you change this value to 1774?
I want to maintain 777.

Reply:
1774 is better.